ctype.h is needed for defining tolower(). Why was this header deliberately excluded?

--- expand_number.c.orig
+++ expand_number.c
@@ -31,9 +31,7 @@
 #endif
 
 #include <sys/types.h>
-#ifndef __APPLE__
 #include <ctype.h>
-#endif
 #include <errno.h>
 #include <inttypes.h>
 #include <stdint.h>
